### PR TITLE
PR Lam Trường

### PR DESCRIPTION
**Pull request by LamTruongJS**
Sumary
- call api
- create add page

### CODE REVIEW

File: src2/App.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  box-sizing: border-box;
+  padding: 0;
+  font-family: "ヒラギノ角ゴシック Pro", "Hiragino Kaku Gothic Pro", "メイリオ",
+    "Meiryo, Osaka", "ＭＳ Ｐゴシック", "MS PGothic", "sans-serif" !important;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
```
Comment (khalt00) [inline src2/App.css:14]:
```suggestion
.App-logo {
  height: 20vmin;
  pointer-events: yes;
}
```
40 is pretty bad, can you change it like this ?

File: src2/App.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
```
[the remark this answers is not in the document]
Comment (LamTruongJS) [reply src2/App.css:31]: Tôi đã fix

[the remark this answers is not in the document]
Comment (LamTruongJS) [reply src2/App.css:31]:
```suggestion
.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}
```

tại sao phải fix như vậy ?

File: src2/Components/Header/HeaderLower/HeaderLower.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+import AddIcon from "@mui/icons-material/Add";
+import { CustomText } from "./style";
+import ModalAppProjectUpper from "../ModalAddProject/ModalAddProject";
+export interface HeaderLowerProps {}
+const HeaderLower: React.VFC<HeaderLowerProps> = ({}) => {
+  const [openModalAddProject, setOpenModalAddProject] = useState(false);
+
+  const handleToggleModalAddProject = () => {
+    setOpenModalAddProject(!openModalAddProject);
+  };
```
Comment (khalt00) [inline src2/Components/Header/HeaderLower/HeaderLower.tsx:14]:
```suggestion
 const useToggleModalAddProject = () => {
    setOpenModalAddProject(!openModal);
  };
```
can you fix it like this?

File: src2/App.css (hunk 1 above)
Comment (LamTruongJS) [reply src2/App.css:14]:
```suggestion
.App-logo {
  height: 40vmin;
  pointer-events: none;
}
```
Is usefull for this project ?

Comment (khalt00) [reply src2/App.css:14]: okay, im sorry

**Pull request merged**